simulator/simulator.hpp

```diff
@@ -82,8 +82,8 @@ namespace mrover {
     struct URDF {
         struct LinkMeta {
             int index{};
-            boost::container::static_vector<Uniform<ModelUniforms>, 2> visualUniforms;
-            boost::container::static_vector<Uniform<ModelUniforms>, 2> collisionUniforms;
+            boost::container::small_vector<Uniform<ModelUniforms>, 2> visualUniforms;
```

Comment:
static vector has limited space and throws if it is exceeded

small vector stores locally but goes to heap if exceeded